RDMA编译问题：   c++: error: ./librdma: No such file or directory, when Linking CXX shared module _swig_paddle.so

这个问题之前没有，今天晚上刚出现。

make过程如下：
cmake .. -DRDMA_ROOT=/root/pp/sxi_for_mlnx -DMKL_ROOT=/opt/intel17/compilers_and_libraries_2017.0.098/linux/mkl -DWITH_DOC=ON -DWITH_GPU=ON -DWITH_AVX=ON -DWITH_RDMA=ON -DWITH_PROFILER=ON
make install


崩在：
[ 95%] Building CXX object paddle/trainer/tests/CMakeFiles/test_recurrent_machine_generation.dir/test_recurrent_machine_generation.cpp.o
Linking CXX executable test_recurrent_machine_generation
Done processing test_recurrent_machine_generation.cpp
Total errors found: 0
[ 95%] Built target test_recurrent_machine_generation
[ 98%] Built target paddle_api
[ 98%] Swig source
Scanning dependencies of target _swig_paddle
[ 98%] Swig source
[ 98%] Building CXX object paddle/api/CMakeFiles/_swig_paddle.dir/PaddlePYTHON_wrap.cxx.o
Linking CXX shared module _swig_paddle.so
c++: error: ./librdma: No such file or directory
make[2]: *** [paddle/api/_swig_paddle.so] Error 1
make[1]: *** [paddle/api/CMakeFiles/_swig_paddle.dir/all] Error 2
make: *** [all] Error 2
[root@g2 build]# 



简单检查一下librdma：
[root@g4 function]# ll /root/pp/git/paddle/build/paddle/function/librdma/
total 0
lrwxrwxrwx 1 root root 30 Mar 26 11:22 libibverbs.so -> /usr/lib64/libibverbs.so.1.0.0
lrwxrwxrwx 1 root root 30 Mar 26 11:22 libibverbs.so.1 -> /usr/lib64/libibverbs.so.1.0.0
lrwxrwxrwx 1 root root 29 Mar 26 11:22 librdmacm.so -> /usr/lib64/librdmacm.so.1.0.0
lrwxrwxrwx 1 root root 29 Mar 26 11:22 librdmacm.so.1 -> /usr/lib64/librdmacm.so.1.0.0
能看到有librdma

这时候，就不知道linker说的那个c++: error: ./librdma: No such file or directory指的是什么了，也不知道怎么找原因。


版本：
[root@g4 build]# git show
commit 929090ed2290b25a1d08959e15a3d6e7a05e1862
Merge: 5961b52 df9993f
Author: QI JUN <qijun1994@hotmail.com>
Date:   Fri Mar 24 18:12:30 2017 +0800

    Merge pull request #1702 from gangliao/python_remove
    
    Fix NOT WITH_PYTHON

另外请教一下，看版本是不是git show   git log？

--- a/paddle/api/CMakeLists.txt
+++ b/paddle/api/CMakeLists.txt
@@ -76,8 +76,6 @@ SWIG_LINK_LIBRARIES(swig_paddle
     ${CMAKE_DL_LIBS}
     ${EXTERNAL_LIBS}
     ${CMAKE_THREAD_LIBS_INIT}
-    ${RDMA_LD_FLAGS}
-    ${RDMA_LIBS}
     ${START_END}
 )
 

